src/axom/CMakeLists.txt

```diff
@@ -39,32 +38,31 @@ axom_add_component(COMPONENT_NAME klee   DEFAULT_STATE ${AXOM_ENABLE_ALL_COMPONE
 axom_add_component(COMPONENT_NAME quest  DEFAULT_STATE ${AXOM_ENABLE_ALL_COMPONENTS})
 axom_add_component(COMPONENT_NAME multimat DEFAULT_STATE ${AXOM_ENABLE_ALL_COMPONENTS})
 
-# Combine all component object libraries into a unified library
-blt_add_library(NAME       axom
-                SOURCES    Axom.cpp
-                DEPENDS_ON ${AXOM_COMPONENTS_ENABLED})
-
-install(TARGETS              axom
+install(TARGETS              ${AXOM_COMPONENTS_ENABLED}
         EXPORT               axom-targets
         DESTINATION          lib)
-install(EXPORT axom-targets DESTINATION lib/cmake)
+install(EXPORT               axom-targets 
+        NAMESPACE            axom::
```

Comment:
What's the effect of this `NAMESPACE axom::` ? Does it add an alias so everything is `axom::<foo>`?

Reply:
All targets added to the `axom-targets` export set will be namespaced by `axom::`. So `fmt`, `core`, and `mpi` will all be `axom::fmt`, etc...

When people include our targets they will end up with `axom`, which depends on everything, and the rest of the targets namespaced under that one. So Serac for example, can just depend on `axom::inlet`.